Sources/Helpers/Factories/ViewCellItemsFactory.swift

```diff
@@ -0,0 +1,160 @@
+    ///
+    /// - Parameters:
+    ///    - CellItem: generated universal cell item or defined in `initializationHandler`  for this object
+    public var viewInitializer: ((CellItem) -> View)? = { _ in
+        .init()
+    }
+
+    // Set this handler to perform primary configuration of`View` after it will be instantiated
+    ///
+    /// - Parameters:
+    ///    - View: an instance  associated with `CellItem`
+    ///    - CellItem: generated universal cell item or defined in `initializationHandler`  for this object
+    public var viewInitialConfigurationHandler: ((View, CellItem) -> Void)?
+
+    // Set this handler to perform view configuration on collection view cell reloading
+    ///
+    /// - Parameters:
+    ///    - View: an instance  associated with `CellItem`
+    ///    - CellItem: generated universal cell item or defined in `initializationHandler`  for this object
+    public var viewConfigurationHandler: ((View, CellItem) -> Void)?
+
+    private lazy var sizeCell: CollectionViewViewCell<View> = {
+        let cell = CollectionViewViewCell<View>()
+        cell.bounds.size = .init(width: 1000, height: 1000)
+        return cell
+    }()
+
+    private lazy var factory: AssociatedFactory = {
+        let factory = AssociatedFactory()
+        factory.cellItemConfigurationHandler = { [weak self] cellItem in
+            guard let cellItem = cellItem as? CellItem else {
+                return
+            }
+            cellItem.sizeTypes = self?.sizeTypesConfigurationHandler?(cellItem.object)
+            cellItem.sizeCell = self?.sizeCell
+            self?.cellItemConfigurationHandler?(cellItem)
+        }
+        factory.cellConfigurationHandler = { [weak self] cell, cellItem in
+            guard let self = self,
+                  let cellItem = cellItem as? CellItem else {
+                return
+            }
+            if let view = cell.view {
+                self.viewConfigurationHandler?(view, cellItem)
+            }
+            else {
+                let view = self.viewInitializer?(cellItem) ?? View()
+                self.viewInitialConfigurationHandler?(view, cellItem)
+                cell.view = view
+                self.viewConfigurationHandler?(view, cellItem)
+            }
+        }
+        factory.initializationHandler = { [weak self] object in
+            if let cellItem = self?.makeUniversalCellItem(object: object) {
+                return [cellItem]
+            }
+            return []
+        }
+        return factory
+    }()
+
+    public lazy var hashKey: String? = .init(describing: Object.self)
+
+    public init() {
+        
+    }
+
+    public func makeCellItems(objects: [Object]) -> [CollectionViewCellItem] {
+        factory.makeCellItems(objects: objects)
+    }
+
+    public func makeDiffCellItems(objects: [Object]) -> [CollectionViewDiffCellItem] {
+        (makeCellItems(objects: objects) as? [CollectionViewDiffCellItem]) ?? []
+    }
```

Comment:
Due to use frequency I would rather rename these functions to 
```
public func makeCellItems(objects: [Object]) -> [CollectionViewDiffCellItem]
```
and 
```
public func makeNonDiffCellItems(objects: [Object]) -> [CollectionViewCellItem]
```
And is it necessary to have this kind of functions split at all?